src/pretix/base/signals.py

```diff
@@ -279,6 +279,13 @@ def connect(self, receiver, sender=None, weak=True, dispatch_uid=None):
 notification settings!
 """
 
+notification = EventPluginSignal()
+"""
+This signal is sent out when a notification is send.
```

Comment:
The documentation is missing information on the parameters that will be available

Reply:
```suggestion
This signal is sent out when a notification is sent.
```